Changes.md

```diff
@@ -12,6 +12,11 @@ Improvements
 - Premultiply, Unpremultiply :
   - Added `ignoreMissingAlpha` plug.
   - Optimised the pass-through of the alpha channel.
+- GraphGadget :
+  - Improved highlighting of active nodes, with more accurate tracking of Loop node iterations.
+  - Annotation `{plug}` substitutions are now evaluated in a context determined relative to the focus node.
+  - The strike-through for disabled nodes is now evaluated in a context determined relative to the focus node.
+- Spreadsheet : Added yellow underlining to the currently active row.
```

Comment:
Looks like this entry has snuck in from the previous PR...

Reply:
Doh! Thanks for spotting that - I've removed it now.